tests/console/docker.pm

```diff
@@ -108,6 +108,15 @@ sub run {
     $output = script_output('docker container run tw:saved curl -I google.de');
     die("network is not working inside of the container tw:saved") unless ($output =~ m{Location: http://www\.google\.de/});
 
+    # docker run supports --init parameter
```

Comment:
I would prefer to remove the comments since the `record_info` is pretty much saying the same thing, but it's up to you.

Reply:
Yes, please remove the comment. Moreover maybe even remove the record_info, if the code doesn't say enough, improve the code, if openQA doesn't show enough of that, improve openQA :)